internal/cmd/marketplace/apply/testdata/withoutErrors/validYaml.yml

```diff
@@ -0,0 +1,41 @@
+---
+_id: 6504773582a6722338be0e25
```

Comment:
I think this property is going to be removed.  
atm we don't need it, so I'll suggest removing it from the test resources.

(same for all the test related resources file)